src/plugins/dashboard/public/application/dashboard_app_controller.tsx

```diff
@@ -494,8 +494,12 @@ export class DashboardAppController {
                 const input = incomingEmbeddable.input;
                 // @ts-expect-error
                 delete input.id;
+                const placeholderTitle = i18n.translate('dashboard.embeddable.placeholderTitle', {
+                  defaultMessage: '[No Title]',
```

Comment:
Is there a use case for defining this in the dashboard controller? Maybe there's something I'm missing, but couldn't this be defined in the panel header so it wouldn't have to passed around so much?

Reply:
Dashboard "decides" what the placeholder title for embeddables on the dashboard will be. Moving this inside panel header would mean that placeholder title for every embeddable would need to be the same. Additionally, we're polluting that component by giving it more responsibility than it should have. It should render the title regardless of where it came from.

Reply:
I see what you mean, but I am still more of the opinion that a placeholder title is exactly the kind of responsibility that the header should have.

It also seems like giving each panel header the same placeholder title would result in greater consistency, and that an addition to  `i_embeddable` is a much more polluting change than adding functionality to `panel_header`